index.js

```diff
@@ -265,19 +265,25 @@ function log (message, site) {
  */
 
 function callSiteLocation (callSite) {
-  var file = callSite.getFileName() || '<anonymous>'
-  var line = callSite.getLineNumber()
-  var colm = callSite.getColumnNumber()
-
-  if (callSite.isEval()) {
-    file = callSite.getEvalOrigin() + ', ' + file
+  var functionName, site
+  if (callSite) {
+    functionName = callSite.getFunctionName()
+    var file = callSite.getFileName() || '<anonymous>'
+    var line = callSite.getLineNumber()
+    var colm = callSite.getColumnNumber()
+    if (callSite.isEval()) {
+      file = callSite.getEvalOrigin() + ', ' + file
+    }
+    site = [file, line, colm]
+  } else {
+    // eslint-disable-next-line no-param-reassign
```

Comment:
I would suggest using a different var name here; the eslint rules are there on purpose :)